libs/libvtrutil/src/vtr_util.h

```diff
@@ -118,6 +118,8 @@ void uniquify(Container container) {
 
 int get_pid();
 
+char* stringf(const char* format, ...);
```

Comment:
This API is actually not really required. The fact is that IMO it is better to make wider use of `std::string` and C++ standards to build strings. I have changed the interchange reader to avoid using this string formatter API